debian/mariadb-server-10.5.mariadb.init

```diff
@@ -26,12 +26,19 @@ SELF=$(cd $(dirname $0); pwd -P)/$(basename $0)
 MYADMIN="/usr/bin/mysqladmin --defaults-file=/etc/mysql/debian.cnf"
```

Comment:
I just copied the existing mysql.init and extended it with some additions improved in Debian so that we now have just one init file and not many incarnations of it with various fixes here and there. I did not rename mysqld or mysqladmin in this PR and will not now fix up after previous renames not done by me to keep the scope manageable.